integrasjon/dokdist-klient/src/main/java/no/nav/foreldrepenger/melding/integrasjon/dokdist/Dokdist.java

```diff
@@ -0,0 +1,9 @@
+package no.nav.foreldrepenger.melding.integrasjon.dokdist;
```

Comment:
Hvorfor beholde den gamle + innføre interface?

Reply:
overgangsording/fallbackmulighet. Men dette initiativet stoppet visst opp @AnjaAalerud ?

Reply:
Ja, det stoppet delvis opp fordi det var oppdaget noen feil med put/patch, og fordi denne klienten bør implementeres i felles i stedet for her siden det er 3 apper som bruker tjenesten. Det ble da ikke prioritert før innvilgelsebrevet.